columnphysics/icepack_aerosol.F90

```diff
@@ -426,8 +427,8 @@ end subroutine update_aerosol
 
 !=======================================================================
 
-!  Increase aerosol in snow surface due to deposition
-!  and vertical cycling : after update_aerosol
+!  Aerosol in snow for vertical biogeochemistry with mushy thermodynamics
+!  Called from icepack_algae.F90 when z_tracers=T (replaces update_aerosol)
 
       subroutine update_snow_bgc (dt,     nblyr,       &
                                 nslyr,                 &
```

Comment:
Looks like everything is in update_snow_bgc, so this should not impact tr_aero unless z_tracers is on?

Reply:
In fact, just tested it out with tr_aero = .true. and NTRAERO=3. It is bfb.